src/infrastructure/dataSchemas/configDataSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { SemanticVersion } from '@/@types/types'
+
+//NOTE: Add any information about StarryDigitizer configurations which is necassary to reconstruct the application states here.
+export interface configDataSchema {
```

Comment:
I believe that the interfaces used for external integration are named 〇〇DataSchema, and those used internally are named 〇〇interface, but I think people who are new to the code might not understand this distinction. It might be better to either change the naming or leave a note somewhere to clarify the usage.

Reply:
Thank you. I will leave a comment about that.

BTW, TS Deep Dive says that we should avoid prefix 'I' on the interface names (which I guess is similar to put suffix 'Interface')
This is one of my top interests about TS and thinking how to avoid conflict between interface names and class name